recipes/clipp/all/conanfile.py

```diff
@@ -1,30 +1,25 @@
 import os
-import glob
 from conan import ConanFile, tools
 
 
+required_conan_version = ">=1.46.0"
```

Comment:
```suggestion
required_conan_version = ">=1.50.0"
```

`self.info.clear()`  https://docs.conan.io/en/latest/changelog.html?highlight=clear#jun-2022